numpy/core/overrides.py

```diff
@@ -12,6 +12,22 @@
 ARRAY_FUNCTION_ENABLED = bool(
     int(os.environ.get('NUMPY_EXPERIMENTAL_ARRAY_FUNCTION', 1)))
 
+array_function_like_doc = (
+    """like : array_like
+        Reference object to allow the creation of arrays which are not
+        NumPy arrays. If an array-like passed in as ``like`` supports
+        the ``__array_function__`` protocol, the result will be defined
+        by it. In this case, it ensures the creation of an array object
+        compatible with that passed in via this argument."""
```

Comment:
```suggestion
        compatible with that passed in via this argument.
        
        .. note::
            
            The ``like`` keyword is an experimental feature pending
            acceptance of `NEP 35 <NEP35>`_
        """
```
Maybe?

Reply:
This is now done in https://github.com/numpy/numpy/pull/16935/commits/bda283cb26c52b7a76fd8686b7afb2f3bf405e38 . However, I don't know if `NEP 35 <NEP35>`_ will render, nor if it's expected to. If there's a different way preferred over this, please let me know.

Reply:
Yeah, I got it wrong, it has to be ``` `:ref:`NEP 35 <NEP35>`  ``` without the trailing underscore but with the ref tag, I guess. We could make it a warning, I did not first because I felt it would distract much from the doc 99% of users will read.

Anyway, others may have opinions on this text?

Reply:
Pushed https://github.com/numpy/numpy/pull/16935/commits/b04df4899556dc298b44c40886240ceaf1329452 to fix the reference.

Reply:
Thanks, this works.  For everyone, the rendered warning is this:

https://15249-908607-gh.circle-artifacts.com/0/doc/build/html/reference/generated/numpy.asarray.html

feedback welcome, although I think fixing it up later is also easy now.